blockchain/fundamentals/intro-blockchain/what-is-the-blockchain.md

```diff
@@ -0,0 +1,62 @@
+---
+author: mihaiberq
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# What Is a Blockchain?
+
+---
+## Content
+
+A **blockchain** (a literal chain of blocks) is a peer-to-peer network designed to build and maintain a sequence of information bundles. 
```

Comment:
Can we add a footnote explaining "peer-to-peer" in casual english?

Reply:
Also, why not say "information blocks" instead of bundles?

Reply:
We might also want to mention that it immutable (cannot be changed)?

The whole analogy about Ebay doesn't show why one would use blockchain over another database for example.

This safety from change is one of the reasons

Reply:
Another aspect is anonimity

Reply:
> Also, why not say "information blocks" instead of bundles?
it's how we described it previously, I didn't change it. I'll update to "blocks"